src/main/java/frc/robot/components/DevilDrive.java

```diff
@@ -65,7 +65,7 @@
   private final CANSparkMax m_frontRightMotor;
   private final CANSparkMax m_rearRightMotor;
   private final double m_deadband;
-  private final double m_maxOutput = 5700;
+  private final double m_maxOutput = 4540.8; //that should be the theoretical max encoder tics per second
```

Comment:
According to [the SparkMax Java docs](https://codedocs.revrobotics.com/java/com/revrobotics/sparkmaxpidcontroller#setReference(double,com.revrobotics.CANSparkMax.ControlType)), the velocity reference value should be in RPM.